apps/web/src/app/(dashboard)/admin/documents/[id]/delete-document-dialog.tsx

```diff
@@ -0,0 +1,122 @@
+'use client';
+
+import { useState } from 'react';
+
+import { useRouter } from 'next/navigation';
+
+import type { Document } from '@documenso/prisma/client';
+import { TRPCClientError } from '@documenso/trpc/client';
+import { trpc } from '@documenso/trpc/react';
+import { Alert, AlertDescription, AlertTitle } from '@documenso/ui/primitives/alert';
+import { Button } from '@documenso/ui/primitives/button';
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from '@documenso/ui/primitives/dialog';
+import { Input } from '@documenso/ui/primitives/input';
+import { useToast } from '@documenso/ui/primitives/use-toast';
+
+export type DeleteDocumentDialogProps = {
+  document: Document;
+};
+
+export const DeleteDocumentDialog = ({ document }: DeleteDocumentDialogProps) => {
```

Comment:
Lets call this `SuperDeleteDocumentDialog` and call the file `super-delete-document-dialog.tsx`, this will make it easier to find the file later on while working in the codebase 😄